.github/workflows/ci.yml

```diff
@@ -84,12 +81,20 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           flavor: |
-            latest=false
+            latest=auto
           images: |
             ghcr.io/banzaicloud/cruise-control
           tags: |
             type=semver,pattern={{version}}
-            type=raw,value=latest,enable={{is_default_branch}}
+            type=edge,branch=master
+          labels: |
+            org.opencontainers.artifact.description=Linkedin's Cruise Control (https://github.com/linkedin/cruise-control)
+            org.opencontainers.image.url=https://github.com/banzaicloud/docker-cruise-control
+            org.opencontainers.image.documentation=https://github.com/banzaicloud/docker-cruise-control
+            org.opencontainers.image.source=https://github.com/banzaicloud/docker-cruise-control"
```

Comment:
Not sure if the `"` at the end of this line is intended

Reply:
nope. fatfingered it. will fix it asap.